js/package.json

```diff
@@ -76,6 +76,7 @@
   "dependencies": {
     "@ai-sdk/provider": "^1.0.1",
     "@braintrust/core": "workspace:*",
+    "@langchain/core": "^0.3.19",
```

Comment:
Should we do this in a separate package within the `js` repo so that all braintrust users do not need to suddenly install langchain? We own the `@braintrust` namespace.

Reply:
💯 

Should we move out the `@ai-sdk/provider` as well?